src/runtime/hexagon/hexagon_device_api.cc

```diff
@@ -154,8 +154,6 @@ void HexagonDeviceAPI::FreeWorkspace(Device dev, void* data) {
   CHECK(runtime_hexbuffs) << "Attempted to free Hexagon workspace with "
                           << "HexagonDeviceAPI::FreeWorkspace outside of a session.  "
                           << "Please call HexagonDeviceAPI::AcquireResources";
-  CHECK(runtime_hexbuffs->count(data) != 0)
```

Comment:
It is ok to remove count, but should be replaced with FindHexagonBuffer.  We want to make sure that the backing data buffer for this workspace is still actively allocated.